### PR TITLE
Add caches for ECS and PHPStan

### PR DESCRIPTION
**Pull request by M-arcus**

### CODE REVIEW

File: .github/workflows/ecs.yml [hunk 1]
```diff
@@ -30,6 +30,13 @@ jobs:
...
+      - name: ECS cache
+        uses: actions/cache@v3
+        with:
+          path: var/cache/cs_fixer
+          key: ${{ runner.OS }}-${{ github.repository }}-ecs-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.OS }}-${{ github.repository }}-ecs-
       - name: Run ECS
         working-directory: custom/plugins/${{ inputs.extensionName }}
         run: ${{ github.workspace }}/vendor-bin/cs-fixer/vendor/bin/ecs check
```
Comment (shyim) [inline .github/workflows/ecs.yml:42]: Can you set here `SHOPWARE_TOOL_CACHE_ECS` as env to `${{ github.workspace }}/var/cs_fixer` and then we cache that folder? It's more stable ;)

File: .github/workflows/phpstan.yml [hunk 2]
```diff
@@ -25,6 +25,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: custom/plugins/${{ inputs.extensionName }}
+      - name: PHPStan cache
```
Comment (shyim) [inline .github/workflows/phpstan.yml:28]: We should do the same here by creating a `phpstan.neon` and including `phpstan.neon.dist` and set a fix
```
parameters:
  tmpDir: PATH
```

Comment (M-arcus) [reply .github/workflows/phpstan.yml:33]: Done

File: .github/workflows/ecs.yml (hunk 1 above)
Comment (M-arcus) [reply .github/workflows/ecs.yml:44]: Done

**Review by shyim — APPROVED** [review]

**Pull request merged**